smac/intensification/successive_halving.py

```diff
@@ -195,6 +200,11 @@ def __init__(self,
         self.fail_challengers = set()  # type: typing.Set[Configuration]
         self.fail_chal_offset = 0
 
+        # Track which configs were launched. This will allow to have an extra check to make sure
+        # that a successive halver deals only with the configs it launched, but also allows querying the
+        # status of the configs via the RH -- knowing what to query
```

Comment:
could you please add a brief explanation why this would have to query the run history in the first place and also write out RH?

Reply:
Improved the comment for this